src/_about/faq.md

```diff
@@ -0,0 +1,142 @@
+---
+title: FAQ
+layout: about-page-faq
+description: Frequently asked questions, for the burning topics that need addressing regularly.
+
+redirect_from:
+- /faq/
+- /faq.html
+---
+
+# General Questions {#general-questions}
+
+{% admonition %}
+
+## What is Quilt?
+
+Quilt is a modding toolchain developed primarily for Minecraft that has been forked from The Fabric Project, primarily 
+due to concerns around governance, project management and moderation. You can read more about the Quilt project and 
+where it came from on [the About Us page](/about).
+
+{% endadmonition %}
+{% admonition %}
+
+## Which parts of Fabric is Quilt forking?
+
+All of them. Quilt is a hard fork of Fabric, and we aim to fork and maintain all projects necessary to exist
+independently of it.
+
+{% endadmonition %}
+{% admonition %}
+
+## Why should I use Quilt?
+
+For users, Quilt provides the following advantages:
+
+* Mods that should be more compatible with each other, despite their use of powerful modding tools
+* Automatic downloading of Quilt's libraries (which can be disabled), so you always have exactly - and only - what you 
+  need for your mods to work
+* Better errors when your mod list has problems with missing or incompatible mods, with more friendly instructions
+* Initial compatibility with both Fabric and Quilt mods, meaning you don't need to stop using your favourites
+* An inclusive, transparent and friendly official community that cares about social issues, has an experienced staff 
+  team, and works with its users to improve over time
+
+{% endadmonition %}
+{% admonition %}
+
+## How can I support Quilt?
+
+If you'd like to help us out, here's just a few ways we've come up with:
+
+* Join [our community spaces](/community), hang out and give your input
+* Contribute to [our projects on GitHub](https://github.com/quiltmc), either directly or by providing reviews and
+  opinions
+* Support other projects that make use of Quilt, or Quilt's tools and libraries
+* Spread the good word on social media (in a responsible manner)
```

Comment:
This sounds too much like a reference to Christianity; please reword

Reply:
I have no idea how someone would think this is related to religion, but I'll see what I can come up with

Reply:
"the good word" sounds 100% like a reference to religion